MAUI/Scheduler/header.md

```diff
@@ -0,0 +1,201 @@
+this.Scheduler.HeaderView.TextFormat = "MMM yy";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+The header can be customized by using the `HeaderView` property of the `SfScheduler.`
+
+### Customize header appearance using text style
+
+You can style the background color, and textStyle by using the properties such as `Background,` and `TextStyle` properties of `HeaderView.`
+
+{% tabs %}
+{% highlight xaml %}
+
+ <scheduler:SfScheduler x:Name="Scheduler" 
+                        View="Week">
+    <scheduler:SfScheduler.HeaderView>
+        <scheduler:SchedulerHeaderView Background="LightGreen">
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+var textStyle = new SchedulerTextStyle()
+{
+    TextColor = Colors.DarkBlue,
+    FontSize = 14,
+};
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.HeaderView.TextStyle = textStyle;
+this.Scheduler.HeaderView.Background = Brush.LightGreen;
+
+{% endhighlight %}  
+{% endtabs %}
+
+### Customize header appearance using DataTemplate
+
+You can customize the header appearance by using the `HeaderTemplate` property of `HeaderView` in the `SfScheduler.` 
+
+{% tabs %}
+{% highlight xaml %}
+
+  <scheduler:SfScheduler x:Name="Scheduler" 
+                         View="Week">
+    <scheduler:SfScheduler.HeaderView>
+        <scheduler:SchedulerHeaderView>
+            <scheduler:SchedulerHeaderView.HeaderTemplate>
+                <DataTemplate>
+                    <Grid Background = "LightGreen">
+                        <Label x:Name="label" TextColor="DarkBlue" HorizontalOptions="Center" VerticalOptions="Center">
+                            <Label.Text>
+                                <MultiBinding StringFormat = "{}{0:MMM dd, yyyy} - {1:MMM dd, yyyy}">
+                                    <Binding Path="StartDate" />
+                                    <Binding Path = "EndDate" />
+                                </MultiBinding>
+                            </Label.Text>
+                        </Label>
+                        <Label  HorizontalOptions="Center" VerticalOptions="End" Text="{Binding Text}" TextColor="Red" />
+                    </Grid>
+                </DataTemplate>
+            </scheduler:SchedulerHeaderView.HeaderTemplate>
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize header appearance using DataTemplateSelector
+
+You can customize the header appearance by using the `HeaderTemplate` property of `HeaderView` in the `SfScheduler.` The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to scheduler header by using the `HeaderTemplate.` It lets to choose a different data template for each header, customizing the appearance of a particular header based on certain conditions.
```

Comment:
It allows you to choose a different data template for each header, as well as to customize the appearance of a particular header based on certain conditions.